src/compiler/crystal/macros.cr

```diff
@@ -149,6 +149,13 @@ module Crystal::Macros
   def skip_file : Nop
   end
 
+  # Returns the current time at compilation  as a unix time stamp.
```

Comment:
```suggestion
  # Returns the current time at compilation (in UTC) as a unix time stamp.
```

Reply:
Unix timestamps are UTC by definition. Is this clarification needed?

Reply:
@jgaskins I don't think that's correct. They're counted since `00:00:00 UTC, 1 January 1970` but they're not necessarily UTC in themselves.

Reply:
If you take it that direction, your clarification is actually incorrect. The number of seconds since that timestamp is the same in all time zones.

Reply:
Well, not necessarily incorrect so much as misleading. It's technically correct, however it's not that many seconds _only_ in UTC, as the clarification appears to imply.

Reply:
You're right, I just wanted to make it crystal clear to the reader :)